ui/app/util.js

```diff
@@ -212,6 +212,7 @@ function readableDate (ms) {
 }
 
 function isHex (str) {
+  if (str.startsWith('0x')) str = str.replace('0x', '')
```

Comment:
I'm not a huge fan of mutating an input in a function intended to answer a simple question about it.

Especially since this is in a general `util` library, this could create unintended consequences for consumers of the method later on (albeit they'd be mitigated by calling `addHexPrefix` on the string afterwards).

I bet there's a way to do all of these things (allow hex, match for either `[g-zG-Z]` or `\W`) all in a single regex, if you were up for the challenge.